src/pages/boundary/api-docs/[[...page]].tsx

```diff
@@ -21,15 +23,21 @@ const targetFile = {
 // The path to read from when running local preview in the context of the boundary repository
 const targetLocalFile = '../../internal/gen/controller.swagger.json'
 
-function ApiDocsView({ layoutProps, apiPageProps }) {
+import { SidebarSidecarLayoutProps } from 'layouts/sidebar-sidecar'
+import type { PageWithLayout } from 'pages/_app'
+
+function ApiDocsLayout(props: SidebarSidecarLayoutProps) {
+	return <SidebarSidecarLayout {...props} />
+}
+
+type ApiDocsViewProps = InferGetStaticPropsType<typeof getStaticProps>
+const ApiDocsView: PageWithLayout<ApiDocsViewProps> = ({ apiPageProps }) => {
 	return (
-		<SidebarSidecarLayout {...layoutProps} sidecarSlot={null}>
-			<OpenApiPageContents
-				info={apiPageProps.info}
-				operationCategory={apiPageProps.operationCategory}
-				renderOperationIntro={apiPageProps.renderOperationIntro}
-			/>
-		</SidebarSidecarLayout>
+		<OpenApiPageContents
+			info={apiPageProps.info}
+			operationCategory={apiPageProps.operationCategory}
+			renderOperationIntro={null}
```

Comment:
I removed `apiPageProps.renderOperationIntro` since that wasn't present, **and** `renderOperationIntro` expects a function, which isn't serializable (via `getStaticProps`) anyways.